app/developers/routes.py

```diff
@@ -705,8 +706,27 @@ def edit_question_validation(grant_id: UUID, question_id: UUID, expression_id: U
     question = get_question_by_id(question_id)
     db_expression = get_expression(expression_id)
 
+    confirm_deletion_form = ConfirmDeletionForm()
+    if (
+        "delete" in request.args
+        and confirm_deletion_form.validate_on_submit()
+        and confirm_deletion_form.confirm_deletion.data
+    ):
```

Comment:
Feels like this route needs a unit test, or are we still in the realms of developer journey so not writing routes tests, just interface tests?